Trouble with inttypes.h when trying to compile a program that uses system inttypes.h on Windows 8 with Visual Studio 2013

When compile a program which uses system inttypes.h on Windows 8 with Visual Studio 2013,
I get these error messages.

```
1>C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\include\inttypes.h(34): error C2371: 'imaxdiv_t' : redefinition; different basic types
1>          c:\workspace\q4\external\sources\libgit2-0.22.2\include\git2\inttypes.h(54) : see declaration of 'imaxdiv_t'
1>C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\include\inttypes.h(36): error C2375: '_abs64' : redefinition; different linkage
1>          C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\include\stdlib.h(388) : see declaration of '_abs64'
1>C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\include\inttypes.h(37): error C2375: 'imaxdiv' : redefinition; different linkage
1>          c:\workspace\q4\external\sources\libgit2-0.22.2\include\git2\inttypes.h(284) : see declaration of 'imaxdiv'
```

--- a/include/git2/common.h
+++ b/include/git2/common.h
@@ -20,7 +20,7 @@
 # define GIT_END_DECL	/* empty */
 #endif
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && _MSC_VER < 1800
  GIT_BEGIN_DECL
 # include "inttypes.h"
  GIT_END_DECL

